### DISCUSSION TITLE
Misplaced egdes (Unaligned edges)

### DISCUSSION THREAD

**Discussion by aryankarim** [bcakmakoglu/vue-flow#1365] [Q&A] [answered]
When using a custom node. All the handles seems to placed at the (0,0) position of the div. Anyone has an idea why is that?

![Screenshot_20240403_222603](https://github.com/bcakmakoglu/vue-flow/assets/58115075/a8ec1c74-fd89-4cee-bc9b-4c4d72876ba5)

**Comment by aryankarim** [comment]
After 7 hours of debugging, I figured it out. It was because the parent element of the VueFlow was a pop-up and had a scaling transition on it.

**Reply by bcakmakoglu** [reply, marked as answer]
Sorry I was already sleeping when you posted this 😅
Yeah I was suspecting that this was the issue.

Problem is an animation that has a scaling transition as you correctly identified. To fix this you can either
1. Hide the Flow until the transition is done
2. Call `updateNodeInternals` when the transition is done
3. Use a transition that does *not* use a transform

**Reply by bcakmakoglu** [reply]
Snippet for what? ^^

**Reply by bcakmakoglu** [reply]
Well solution 1. and 3. is up to you, I have no code-snippets for these.
You can hide a flow with css or v-show or whatever you wanna use to hide it.
Using a different transition is also not something I can show you a code snippet for - I don't know what Modal you use etc., that's for you to figure out 😄 

[As for 2. - there's a section in the docs for it.](https://vueflow.dev/guide/node.html#working-with-dynamic-handle-positions-adding-handles-dynamically)
